src/components/navbar/Navbar.vue

```diff
@@ -168,10 +168,16 @@ export default {
             ])
         },
         genNavbarBrandNode() {
+            let children
```

Comment:
Semicolon?

Reply:
Does this look better?
```js
const children = this.$slots.brand != null
    ? [this.$slots.brand(), this.genBurgerNode()]
    : this.genBurgerNode()
```

Reply:
Yes that works ✨

Reply:
You can also add parentheses to make it clear what expression is being used in the condition of the ternary operator

```javascript
const children = (this.$slots.brand != null) 
       ? [this.$slots.brand(), this.genBurgerNode()] 
       : this.genBurgerNode()
```